src/planwise/endpoint/home.clj

```diff
@@ -103,6 +103,10 @@
       (include-js "/js/leaflet.bboxloader.js")
       (include-js "/js/leaflet.legend.js")
       (include-js "/js/main.js")
+      (include-js "/js/leaflet.markercluster.js")
+      (include-css "/css/MarkerCluster.css")
+      (include-css "/css/MarkerCluster.Default.css")
+      (include-css "/css/MarkerCluster.Custom.css")
```

Comment:
CSS should be included from the header, along with the other CSS files. And we should probably integrate these files into the SASS compilation.